contrib/intel/jenkins/build.py

```diff
@@ -0,0 +1,193 @@
+import os
+import sys
+
+# add jenkins config location to PATH 
+sys.path.append(os.environ['CI_SITE_CONFIG'])
+
+import ci_site_config
+import argparse
+import subprocess
+import shlex
+import common
+
+#read environment variables
+branchname = os.environ['BRANCH_NAME']
+buildno = os.environ['BUILD_NUMBER']
+workspace = os.environ['WORKSPACE']
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--builditem", help="build libfabric or fabtests",
+                     choices=['libfabric','fabtests'])
+parser.add_argument("--mpi", help="select mpi type for building mpi benchmarks",
+                    choices=['impi', 'ompi', 'mpich'])
+parser.add_argument("--build_mode", help="select buildmode debug or dl", choices=['dbg','dl'])
+parser.add_argument("--other_benchmarks", help="build other non mpi benchmarks",
+                           choices=['shmem'])
+
```

Comment:
The above code can be within a `if __name__ == "__main__":` block to allow for importing this file as a module in the future (if need arises).